### PR TITLE
[WIP] Tests on arm01

### PR DESCRIPTION
**Pull request by danilo-gemoli**
Follow up of https://github.com/openshift/release/pull/41200
/cc @SaravanaStorageNetwork @deepsm007 @droslean

### CODE REVIEW

**Comment by danilo-gemoli** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.13-ocp-e2e-aws-ovn-arm64-techpreview

File: ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.14.yaml [hunk 1]
```diff
@@ -130,6 +130,7 @@ resources:
       memory: 200Mi
 tests:
 - as: ocp-e2e-ibmcloud-ovn-multi-s390x
+  cluster: arm01
```
Comment (SaravanaStorageNetwork) [inline ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.14.yaml:133]: This is for s390x architecture and not ARM.

File: ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.15.yaml [hunk 2]
```diff
@@ -130,6 +130,7 @@ resources:
       memory: 200Mi
 tests:
 - as: ocp-e2e-ibmcloud-ovn-multi-s390x
+  cluster: arm01
```
Comment (SaravanaStorageNetwork) [inline ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.15.yaml:133]: This is for s390x architecture.

**Comment by danilo-gemoli** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.13-ocp-e2e-aws-ovn-arm64-techpreview

**Comment by danilo-gemoli** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.13-ocp-e2e-aws-ovn-arm64-techpreview

**Comment by danilo-gemoli** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.13-ocp-e2e-aws-ovn-arm64-techpreview

File: ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.10.yaml [hunk 3]
```diff
@@ -128,6 +129,7 @@ tests:
       TEST_ARGS: --from-repository quay.io/multi-arch/community-e2e-images
     workflow: openshift-e2e-aws
 - as: ocp-installer-e2e-aws-ovn-arm64
+  cluster: arm01
```
Comment (deepsm007) [inline ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.10.yaml:132]: dependencies should be `RELEASE_IMAGE_LATEST: release:arm64-latest`

**Comment by danilo-gemoli** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.13-ocp-e2e-aws-ovn-arm64-techpreview

**Comment by danilo-gemoli** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.13-ocp-e2e-aws-ovn-arm64-techpreview

**Comment by danilo-gemoli** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.13-ocp-e2e-aws-ovn-arm64-techpreview

**Comment by danilo-gemoli** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.13-ocp-e2e-aws-ovn-arm64-techpreview

**Pull request closed without merging**